include/cantera/base/Solution.h

```diff
@@ -8,14 +8,13 @@
 
 #include "cantera/base/ctexceptions.h"
 #include "cantera/base/AnyMap.h"
+#include "cantera/thermo/ThermoPhase.h"
+#include "cantera/kinetics/Kinetics.h"
+#include "cantera/transport/TransportBase.h"
```

Comment:
Adding these here means that this file will have to be recompiled any time one of these dependencies changes. It's a small change to compiling times probably, but I don't think it's necessary here.

Reply:
Removed. Going with a new header file `cantera/core.h` instead.